client/app/hearings/components/ListSchedule.jsx

```diff
@@ -157,8 +157,8 @@ class ListSchedule extends React.Component {
     this.setState({ dateRangeKey: `${this.props.startDate}->${this.props.endDate}` });
   }
 
-  formatHearingsScheduled = (filledSlots, totalSlots) => {
-    return `${filledSlots} of ${totalSlots}`;
+  formatHearingsScheduled = (filledSlots) => {
+    return `${filledSlots}`;
```

Comment:
No need for string interpolation anymore
```suggestion
    return filledSlots;
```